addons/account_edi/wizard/account_invoice_send.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+import base64
+
+
+class AccountInvoiceSend(models.TransientModel):
+    _inherit = 'account.invoice.send'
+    _description = 'Account Invoice Send'
+
+    include_xml = fields.Boolean('XML')
+    xml_ids = fields.Many2many('ir.attachment', 'account_invoice_send_ir_attachments_rel', 'wizard_id', 'attachment_id')
```

Comment:
`compute='_compute_xml_ids'`

```
@api.depends('include_xml')
def _compute_xml_ids(self):
    ....
```

or remove `xml_ids` and transform `attachment_ids` as an overridable computed field